### PR TITLE
Submission revisions

### PR DESCRIPTION
**Pull request by emmetfrancis**
Associated changes:
- Update README with latest information on downloading meshes and data, add Zenodo badges.
- Update README in scripts to include all scripts to run, add information on SMART with new tag to be generated soon (v2.2.2)
- Update plot generation for dendritic spine timings - include individual data points in stacked bar graphs and correct the formulae for standard deviations.
- Update TSCC bash scripts to match the final runs for the paper.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.11268944.svg)](https://doi.org/10.5281/zenodo.11268944)
+Meshes: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.10480303.svg)](https://doi.org/10.5281/zenodo.10480303)
+Results: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.11252054.svg)](https://doi.org/10.5281/zenodo.11252054)
+
```
Comment (jorgensd) [inline README.md:4]:
```suggestion
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.11268944.svg)](https://doi.org/10.5281/zenodo.11268944)

Meshes: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.10480303.svg)](https://doi.org/10.5281/zenodo.10480303)

Results: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.11252054.svg)](https://doi.org/10.5281/zenodo.11252054)

```
Maube each should render on a new line?

**Review by jorgensd — APPROVED** [review]
LGTM! 👍🏻 
I added some minor changes and fixed some typos.
Only one stylistic question regarding the badges left as a comment. Let me know what you think.

**Review by finsberg — COMMENTED** [review]
I think we should make a new release of `smart` (e.g v0.3.0) and use this as the target version rather than the development branch. It might be that the code here will stop working with the development version in the future.

Also I wonder wether we should add a tests to check that all commands work. I can add a simple test and CI to check this, e.g
```python
import pytest
from main import commands

@pytest.mark.parametrize("cmd", commands.values(), ids=list(commands.keys()))
def test_scripts_dry_run(cmd):
    """Just make sure all scripts work with dry run"""
    # Just add some dummy arguments
    args = {k:"" for k in cmd.script.__code__.co_varnames if k != "dry_run"}
    cmd.script(dry_run=True, **args)
```

File: README.md (hunk 1 above)
Comment (emmetfrancis) [reply README.md]: Agreed, thanks!

**Comment by emmetfrancis** [comment]
@jorgensd do you think this is good for merging?

**Comment by jorgensd** [comment]
> @jorgensd do you think this is good for merging?

Yes

**Pull request merged**